### PR TITLE
Add extract-zip types

### PR DESCRIPTION
**Pull request by mizunashi-mana**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.

### CODE REVIEW

File: extract-zip/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
```
Comment (andy-ms) [inline extract-zip/tsconfig.json]: Needs `"noImplicitThis": true,` below this line.

Comment (mizunashi-mana) [inline extract-zip/tsconfig.json]: Sorry, I done.

**Comment by andy-ms** [comment]
We don't accept pull requests to `types-2.0` any more (#13233). Please rebase onto `master`, then hit "Edit" in the upper-right and change the base branch to `master`.

**Comment by mizunashi-mana** [comment]
@andy-ms I see. done.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**